TESTS/netsocket/udp/udpsocket_recv_timeout.cpp

```diff
@@ -27,6 +27,7 @@ using namespace utest::v1;
 namespace {
 static const int SIGNAL_SIGIO = 0x1;
 static const int SIGIO_TIMEOUT = 5000; //[ms]
+static const int PKT_NUM = 10;
```

Comment:
After a more though on this.. Actually, not even sure why we repeat this 10 times.
Maybe 2 would be enough.

After all the idea of this test case is to test timeouts, not actually receiving of the packets.

Reply:
Good point, I decreased the number to 2.